libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragment.java

```diff
@@ -833,11 +833,12 @@ public void run() {
     }
 
     @Override
-    public void onMediaUploadFailed(final String mediaId) {
+    public void onMediaUploadFailed(final String mediaId, final String errorMessage) {
         mWebView.post(new Runnable() {
             @Override
             public void run() {
-                mWebView.execJavaScriptFromString("ZSSEditor.markImageUploadFailed(" + mediaId + ");");
+                mWebView.execJavaScriptFromString("ZSSEditor.markImageUploadFailed(" + mediaId + ", '"
+                        + errorMessage + "');");
```

Comment:
:facepalm:, yes we need to quote all strings coming from Java, especially this one that come from the server.